addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/__tests__/environment-sc-service.test.js

```diff
@@ -820,4 +820,142 @@ describe('EnvironmentSCService', () => {
       );
     });
   });
+
+  describe('getSecurityGroupDetails function', () => {
```

Comment:
Probably add a scenario on how we handle cases where we don't find a securityGroupIngress. Sounds like an Error case instead of getting empty, right?

Reply:
Yes, I'll add that check with a test case. Thanks.

Reply:
Done